.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/)
+- [ ] I viewed the changes on the sev server (`yarn dev`) to confirm they render without error
+- [ ] The terms used in this change correspond to those in [Teleport Core Concepts](https://goteleport.com/docs/core-concepts)
+- [ ] I spellchecked the changes
```

Comment:
Hopefully we won't need this if we merge this PR: https://github.com/gravitational/teleport/pull/23783